src/config.c

```diff
@@ -260,6 +291,95 @@ void configureVifs(void) {
     }
 }
 
+/* create VIFs for all IP, non-loop interfaces.
+   When argument is not NULL rebuild the interface table.
+*/
+void createVifs(struct IfDescP *RebuildP) {
+    struct IfDesc *Dp, *oDp = NULL;
+    int    vifcount = 0, upsvifcount = 0, Ix = 0;
+    bool   join = 0;
```

Comment:
`false` and `true` are bool values. So either use `int join = 0;` or `bool join = false`.

Reply:
fixed